app/src/androidTest/java/com/android/voyageur/ui/profile/ProfileScreenTest.kt

```diff
@@ -160,4 +169,31 @@ class ProfileScreenTest {
     // Assert: Verify that the navigation was triggered
     verify(navigationActions).navigateTo(Route.AUTH)
   }
+
```

Comment:
You could also add a test which has null or white spaces as interest and check that it doesn't get saved (I checked manually and it works).

Reply:
Thank you very much for the suggestion!

Reply:
I noticed you added the test, thanks!